package.json

```diff
@@ -77,7 +77,7 @@
                     "type": "number",
                     "description": "The penalty value used by the BLOCK strategy",

```

Comment:
I was just scratching my head at this setting. Would be nice if this setting could be hidden unless `BLOCK` strategy is set, but not sure if that's possible. All these bottleneck settings could probably use a link to https://www.npmjs.com/package/bottleneck to clarify their purpose. 

I'd update the description to this setting to be a little clearer:
>Specific to the "BLOCK" strategy, if another strategy is used this setting is ignored.
>
>The time, in milliseconds, that new jobs are dropped when the command queue reaches the high water mark. The limiter will unblock after **penalty** seconds have passed without receiving a new job. See https://www.npmjs.com/package/bottleneck#bottleneckstrategyblock

Reply:
thanks for commenting! I'll tweak the descriptions a bit more. 

I was tempted to hide most of these settings as it's generally not something that an end user would want to tweak, but was reading the justifcations in https://github.com/stef-levesque/vscode-perforce/issues/137 I decided to leave them in.

I don't think it's possible to selectively show settings, though possibly json schema can be applied so that it's not valid to set one without the other.

As a side note I definitely think bottleneck could use some more precise / detailed documentation as it took me a while to get my head around it, and I'm not sure that it was understood when implemented in the first place - the existing defaults set the max high water to 50 but the max concurrent is unlimited - as far as I can tell, the high water will never increase above 0 with those settings.